core/types/deposit_tx.go

```diff
@@ -24,6 +24,8 @@ import (
 
 const DepositTxType = 0x7E
 
+var zeroBig = new(big.Int)
```

Comment:
The `big` package has a global for this already

Reply:
I use `common.Big0` for this